inst/metadata/colour_scales.csv

```diff
@@ -22,6 +22,8 @@ default,anc_art_new,interpolateOranges,0,350000,FALSE
 default,anc_known_pos,interpolateOranges,0,350000,FALSE
 default,anc_tested_pos,interpolateOranges,0,350000,FALSE
 default,anc_tested_neg,interpolateOranges,0,700000,FALSE
+default,anc_prevalence_age_matched,interpolateOranges,0,0.5,FALSE
```

Comment:
I think these colour don't actually ever get used as these indicators are only in barchart but we need to add it to satisfy some of the tests